### PR TITLE
Update stagedsync readme to show all current stages

### PR DESCRIPTION
**Pull request by The-Coomer**
Hi, I have updated the readme to list all the current 18 sync stages. 

I was confused when first syncing erigon with stages that didn't show up on here, when trying to get an idea about how far into the process I was.

### CODE REVIEW

File: eth/stagedsync/README.md [hunk 1]
```diff
@@ -106,7 +118,17 @@ This stage is disk intensive.
 
 This stage can spawn unwinds if the block execution fails.
 
-### Stage 6: [Generate Hashed State Stage](/eth/stagedsync/stage_hashstate.go)
+### Stage 8: [Transpile marked VM contracts to TEVM](/eth/stagedsync/stage_tevm.go)
+
+[TODO]
```
Comment (mandrigin) [inline eth/stagedsync/README.md:123]: I think this one also is not enabled by default.

**Review by mandrigin — APPROVED** [review]
Thanks a lot!

**Comment by AskAlexSharov** [comment]
Actually Key to success here is - to "not use numbers" because we can add stage in the middle, re-org stages, etc... Stages have name and better use only names. 

Merging. Thank you.

File: eth/stagedsync/README.md (hunk 1 above)
Comment (AskAlexSharov) [reply eth/stagedsync/README.md:123]: nope

**Pull request merged**